data/frm/Crecy_1997_xPonthus/stemma.gv

```diff
@@ -0,0 +1,23 @@
+digraph {
+
+x [color="grey"];
+x -> P1; 
+x -> P2; 
+x -> P3; 
+x -> P4; 
+
+x-> 1;
+1 [label="", color="grey"];
+1 -> G; 
+1 -> P5;
+
+x -> 2;
+2 [label="", color="grey"];
+2 -> CF; 
```

Comment:
Descendants not shown on the stemma, where are they from ? Described in the text ?

Reply:
Yes, they are described in the text.